docs/dep.md

```diff
@@ -1,40 +1,32 @@
 # kubernetes/test-infra dependency management
 
-test-infra uses [`dep`](https://github.com/golang/dep) for Go dependency
-management. `dep` is a prototype dependency management tool for Go. It requires
-Go 1.8 or newer to compile.
+test-infra uses [`dep`] for Go dependency
+management. Usage requires [bazel], which can be accessed
+through [`planter`] if not locally installed.
 
+## Usage
 
-## Setup
+Run [`hack/update-deps.sh`] whenever vendored dependencies change. This will
+take around 5m to complete.
 
-You can follow the [setup instructions](https://github.com/golang/dep#setup) to
-set up `dep` in your local environment.
+### Advanced usage
 
-test-infra is currently using dep 0.4.1. This version has also been vendored
-into the repository. You can run the vendored version using `bazel run`; for
-example, run
-```console
-$ bazel run //:dep -- ensure -add example.org/foo/bar
-```
-to add a new dependency.
-
-## Changing dependencies
-
-You can use the `dep` instructions for
-[adding](https://golang.github.io/dep/docs/daily-dep.html#adding-a-new-dependency),
-[updating](https://golang.github.io/dep/docs/daily-dep.html#updating-dependencies) or
-[removing](https://golang.github.io/dep/docs/daily-dep.html#using-dep-ensure) a dependency.
-
-Once you've updated, make sure to run:
-```
-hack/update-deps.sh
-```
-This will generate necessary Bazel BUILD files and will also remove any
-unused libraries that dep fails to prune.
+* [add] - Run `hack/update-deps.sh --add cloud.google.com/go/storage@v0.17.0 [...]` to pin a dependency at a particular version
+* [update] - Run `hack/update-deps.sh --update golang.org/x/net [...]` to update an existing dependency
+* [remove] - Edit `Gopkg.toml` and run `hack/update-deps.sh` to remove or unpin dependencies
 
 ## Tips
 
 If `dep ensure` doesn't come back and freezes, please make sure `hg` command is
 installed on your environment. `dep ensure` requires `hg` command for getting
 bitbucket.org/ww/goautoneg , but `dep ensure` doesn't output such error message
-and just freezes. [reference](https://github.com/kubernetes/test-infra/issues/5987)
+and just [freezes]
```

Comment:
nit: please add back the trailing period. :)

Reply:
:roll_eyes:

Reply:
Done